include/mgp.hpp

```diff
@@ -2645,6 +2648,8 @@ inline void Relationship::SetProperty(std::string property, Value value) {
   mgp::edge_set_property(ptr_, property.data(), value.ptr());
 }
 
+inline void Relationship::RemoveProperty(std::string property) { SetProperty(property, Value()); }
```

Comment:
Is it possible to pass here only `{}` as an empty value

Reply:
No:
`Chosen constructor is explicit in copy-initialization`